mobile/lib/modules/home/views/home_page.dart

```diff
@@ -47,11 +47,11 @@ class HomePage extends HookConsumerWidget {
 
     useEffect(
       () {
-        ref.watch(websocketProvider.notifier).connect();
-        ref.watch(assetProvider.notifier).getAllAsset();
-        ref.watch(albumProvider.notifier).getAllAlbums();
-        ref.watch(sharedAlbumProvider.notifier).getAllSharedAlbums();
-        ref.watch(serverInfoProvider.notifier).getServerVersion();
+        ref.read(websocketProvider.notifier).connect();
+        Future(() => ref.read(assetProvider.notifier).getAllAsset());
```

Comment:
Why `Future` is used here?

Reply:
getAllAssets is called during build but can modify state causing a rebuild, flutter throws an error/warning and suggest to put the call in a future so it's delayed after painting